docs/source/tutorials/configure_systems.rst

```diff
@@ -345,7 +382,7 @@ Ort Environment System
                 "type": "IsolatedORT",
                 "config": {
                     "python_environment_path": "/home/user/.virtualenvs/myenv/bin",
-                    "accelerators": ["cpu"]
+                    "accelerators": [{"device": "cpu"}]
```

Comment:
Can you also add ep in this example to make it consistent with the above systems? 
I also forgot to update the title at line 374. Can you correct that too? Thanks!

Reply:
will do